testing/unit_tests/test_loader.py

```diff
@@ -1,17 +1,16 @@
 import pytest
 import csv_diff
-import logging
 import torch
 import numpy as np
 
 from ivadomed.loader.bids_dataframe import BidsDataframe
 from testing.unit_tests.t_utils import create_tmp_dir, __data_testing_dir__, __tmp_dir__, download_data_testing_test_files, path_repo_root
 from testing.common_testing_util import remove_tmp_dir
-from ivadomed.loader import utils as imed_loader_utils
 from ivadomed.loader import loader as imed_loader
+import ivadomed.loader.utils as imed_loader_utils
 from ivadomed.keywords import MetadataKW
 from pathlib import Path
-logger = logging.getLogger(__name__)
+from loguru import logger
```

Comment:
Switching over to the `loguru` logger.

Reply:
I think `logger` is not used in this file anymore. We should remove this.

Reply:
Good point.